code/modules/power/groundmap_geothermal.dm

```diff
@@ -589,6 +598,64 @@ GLOBAL_VAR_INIT(active_bluespace_generators, 0)
 	icon_state = "circ-on75-cold"
 
 
+/// Psychic mist -- Spawns on generator if it explodes; provides point gen as the generator would have
+/obj/machinery/mist_origin
+	name = ""
+	resistance_flags = RESIST_ALL|PROJECTILE_IMMUNE|DROPSHIP_IMMUNE
```

Comment:
Why is this a machine and not an effect

Reply:
Effects don't have process()

Reply:
update: moving it to an effect because processing can be moved to gamemode process